migrations/2022-06-15-safe-deleted.js

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env node
+/* eslint no-await-in-loop: off */
+require('dotenv').config()
+const mongo = require('../lib/util/mongo')
+
+async function main() {
+  await mongo.connect()
+
+  const modifiedBALs = await mongo.db.collection('bases_locales').updateMany(
+    {_deleted: {$exists: false}},
+    {$set: {_deleted: null}}
+  )
+
+  console.log('\nTerminé !\n')
+  console.log('************************************\n')
+  console.log(` ${modifiedBALs.modifiedCount} BAL ont été modifiées \n`)
+  console.log('************************************\n')
+
+  await mongo.disconnect()
+  process.exit(1)
```

Comment:
```suggestion
  process.exit()
```
le code `1` est utilisé en cas d'erreur